### ISSUE TITLE
No redundand fallback defaults in templates if they are already in defaults/main.yml

### ISSUE THREAD

**Issue by ypid** [debops/debops-playbooks#310]
Example: ["nginx_hsts_age | default('15768000')" in `debops.nginx`](https://github.com/debops/ansible-nginx/blob/debcc2b66f8afede7ba87e41ad24b3d1753bbb72/templates/etc/nginx/sites-available/default.conf.j2#L1061)

I have seen that a number of times in DebOps but I think it does not add much value instead maintenance overhead. Thoughts?

**Comment by drybjed** [comment]
Agreed. Defaults in templates aren't easily changed by the user, so we can drop them and rely on the variables being present in `defaults/main.yml`. In the specific `debops.nginx` case and `nginx.conf` template I was trying to make the number of the variables in defaults smaller, hoping that the user could add their overrides for variables set there through the inventory anyway. But I guess that I just need to get used to long `defaults/main.yml` files instead. :-)

**Comment by ypid** [comment]
All right. I guess you can set this to "approved" then :wink: 

> But I guess that I just need to get used to long defaults/main.yml files instead. :-)

Thats what we have tables of contents in rendered docs and [Vim folds](https://github.com/debops/docs/issues/156) for the raw files for :smile:

**Comment by drybjed** [comment]
@ypid Yup, I know. I wasn't using them back then. :-)
